wazo_agid/handlers/tests/test_userfeatures.py

```diff
@@ -54,27 +54,25 @@ def test_userfeatures(self):
         self.assertEqual(userfeatures._cursor, self._cursor)
         self.assertEqual(userfeatures._args, self._args)
 
-    def test_set_members(self):
+    @patch('wazo_agid.objects.User')
+    def test_set_members(self, mock_user: Mock):
```

Comment:
Objection to use the following syntax to avoid confusion of unused variable?
```suggestion
    @patch('wazo_agid.objects.User', Mock())
    def test_set_members(self):
```

Reply:
If you think it could cause confusion, sure. Usually, you want to receive it since you want to test for calls on it. However, this test doesn't do that, so I suppose it isn't required. We can change it back if we decide to test more thoroughly in future. :)